optimize(getting started): move install instructions to first page (#1718)
timescale/docs: getting-started/index.md

excerpt: Get started with your first TimescaleDB instance
---

import Install from "versionContent/_partials/_cloud-installation.mdx";
import CreateService from "versionContent/_partials/_cloud-create-service.mdx";
import Connect from "versionContent/_partials/_cloud-connect.mdx";

# Getting started with TimescaleDB

Get started with TimescaleDB to experience the power of its core features, such
as hypertables, continuous aggregates, and compression.

## What is TimescaleDB?

TimescaleDB is an extension on top of PostgreSQL. It gives you all the power of
PostgreSQL, plus new superpowers that help you work with time-series data and
complex SQL queries.

> ## About the getting-started guide

Reviewer: ```suggestion
## About this guide
```

> In this tutorial, you'll set up a TimescaleDB database and work with some

Reviewer: ```suggestion
This guide helps you set up a TimescaleDB database, so you can work with some
```
